src/styles/utils/variables.scss

```diff
@@ -0,0 +1,11 @@
+$start-day: ('mon' 0, 'tue' 1, 'wed' 2, 'thu' 3, 'fri' 4, 'sat' 5, 'sun' 6);
+$cellSize: 100px;
+$cellCollor: #eee;
+$hoverCollor: #ffbfcb;
+$cellFontHeight: 30px;
+$cellFontFamily: Arial, sans-serif;
+$gap: 1px;
+$weakSize: 7;
```

Comment:
The variable `$weakSize` seems to be incorrectly named. If it represents the number of days in a week, it should be named `$weekSize` to avoid confusion.